cheatsheet.js

```diff
@@ -0,0 +1,300 @@
+console.log(10 == 20);         // Log: false
+console.log('asdf' == 'asdf'); // Log: true
+
+// The `===` operator checks for *type safe* equality and evals to `true` or `false`.
+// When using `==`, Javascript will try to convert each argument to the same type.
+console.log('10' == 10);       // Log: true
+console.log('10' === 10);      // Log: false
+console.log('asdf' == true);   // Log: true
+console.log('asdf' === true);  // Log: false
+console.log('' == false);      // Log: true
+console.log('' === false);     // Log: false
+console.log(1 == true);        // Log: true
+console.log(1 === true);       // Log: false
+console.log(0 == false);       // Log: true
+console.log(0 === false);      // Log: false
+
+// Fun Fact:
+// Every value in Javascript can be converted, or "cast" to any other type, to differing effect.
+// You can test these conversions yourself in the browser by using the "primitive constructors"
+// `String`, `Boolean`, and `Number`, to cast values to different types and see what comes out.
+// For example:
+String(10);    // "10"
+Boolean(245);  // true
+Boolean("");   // false
+Number("100"); // 100
+
+// The `<=` operator checks if the left value is *Less Than or Equal To* the right value.
+console.log(10 <= 5);  // Log: false
+console.log(5 <= 10);  // Log: true
+console.log(10 <= 10); // Log: true
+
+// The `>=` operator checks if the left value is *Greater Than or Equal To* the right value.
+console.log(10 >= 5);  // Log: true
+console.log(5 >= 10);  // Log: false
+console.log(10 >= 10); // Log: true
+
+// The `<` operator checks if the left value is *Less Than* the right value.
+console.log(10 < 5);  // Log: false
+console.log(5 < 10);  // Log: true
+console.log(10 < 10); // Log: false
+
+// The `>` operator checks if the left value is *Greater Than* the right value.
+console.log(10 > 5);  // Log: true
+console.log(5 > 10);  // Log: false
+console.log(10 > 10); // Log: false
+
+// The `||` operator returns `true` or `false` if the left side *OR* the right side is truthy
+console.log(true || false);          // Log: true
+console.log(false || false);         // Log: false
+console.log( (10 > 5) || (10 < 5) ); // Log: true
+
+// The `&&` operator returns `true` or `false` if the left side *AND* the right side is truthy
+console.log(true && false);        // Log: false
+console.log(true && true);         // Log: true
+console.log((10 > 5) && (10 < 5)); // Log: false
+
+/***************** Math Expressions *******************/
+// Math expressions will return a new value depending on the input values
+// The most common input for these operators are number values.
+
+// + Addition Operator (Used to concatonate strings too!)
+console.log(1 + 2) // Log 3
+console.log('Trevor' + ' Foehl') // Log "Trevor Foehl"
+
+// - Subtraction Operator
+console.log(3 - 2) // Log 1
+
+// * Multiplication Operator
+console.log(2 * 2) // Log 4
+
+// \ Division Operator
+console.log(2 / 2) // Log 1
+
+
```

Comment:
@amiller-gh Add in modulo here?
Modulo (written using %) is basically the remainder:

30 / 3 = 10, remainder 0
=> `30 % 3 = 0`

10 / 3 = 3, remainder 1
=> `10 % 3 = 1`

Reply:
Ooh, good call!